private/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package private contains helpers private to github.com/containers/image, with no intent or commitment to make them useful for any other purpose.
+// Do not use outside!
+package private
```

Comment:
so, I'd rather have a `pkg/MEANINGFULNAME` for each stuff in this package instead of a catch-all-oo-utils-like packge